js/helpers/email.js

```diff
@@ -0,0 +1,26 @@
+import { htmlTemplate } from './emailtemplate';
+
+export const contentValueString = (data, fname, lname) => {
+  return `Hello ${fname}${lname} how are you? i hope you are doing well. I am doing fine thank you from ${data.sender}`
+};
+
+export function sendEmail(data) {
+  fetch('https://api.sendgrid.com/v3/mail/send', {
```

Comment:
Maybe spin the email generation out to a service with the url being passed via a function.